lib/index_styling.scss

```diff
@@ -0,0 +1,56 @@
+td.green {
```

Comment:
Another potential refactor + something to be aware of once we merge our code bases --  let's use class names or id for styling, I think using the elements here might overwrite any the styling/formatting for my tables too.